stdlib/_typeshed/__init__.pyi

```diff
@@ -167,8 +168,13 @@ class SupportsNoArgReadline(Protocol[_T_co]):
 class SupportsWrite(Protocol[_T_contra]):
     def write(self, __s: _T_contra) -> Any: ...
 
-ReadableBuffer = Union[bytes, bytearray, memoryview, array.array[Any], mmap.mmap]  # stable
-WriteableBuffer = Union[bytearray, memoryview, array.array[Any], mmap.mmap]  # stable
+ReadOnlyBuffer = bytes  # stable
+# Anything that implements the read-write buffer interface.
+# The buffer interface is defined purely on the C level, so we cannot define a normal Protocol
+# for it. Instead we have to list the most common stdlib buffer classes in a Union.
+WriteableBuffer = Union[bytearray, memoryview, array.array[Any], mmap.mmap, ctypes._CData]  # stable
+# Same as _WritableBuffer, but also includes read-only buffer types (like bytes).
```

Comment:
```suggestion
# Same as _WriteableBuffer, but also includes read-only buffer types (like bytes).
```

Reply:
Nice catch, thanks. This was not intentional, and probably when I started wondering about its spelling.